layouts/generic-py-base.html

```diff
@@ -31,10 +31,8 @@
         </div>
       </div>
       {% block global_subheader_container %}
-        <div class="global-subheader">
-          <div class="container">
-            {% block global_subheader %}{% endblock %}
-          </div>
+        <div class="global-subheader container">
```

Comment:
Also as a single div in the documentation. Maybe it needs to be updated instead?